exercises/isbn-verifier/description.md

```diff
@@ -15,7 +15,7 @@ The following number block is to identify the publisher. Since this is a three d
 The last digit in the ISBN is the check digit which is used to detect read errors.
 
 The first 9 digits in the ISBN have to be between 0 and 9.
-The check digit can additionally be an 'X' to allow 10 to be a valid check digit as well.
+The check digit can additionally be a 'X' to allow 10 to be a valid check digit as well.
```

Comment:
@christophschlosser my apologies.  I was not clear on when `an` should be `a`.  In this instance `an` is OK.  An is used when the next word starts with a vowel or starts with a vowel sound.  In this instance 'X' starts with a vowel sound, `e` in this instance.

Reply:
No problem @rpottsoh . TIL. I uploaded a new commit where this is fixed.